samanthas_telegram_bot/api_clients/backend/client.py

```diff
@@ -65,6 +67,11 @@ async def chat_id_is_registered(cls, update: Update, context: CUSTOM_CONTEXT_TYP
 
         return status_code == httpx.codes.OK
 
+    @classmethod
+    async def create_coordinator(cls, update: Update, context: CUSTOM_CONTEXT_TYPES) -> bool:
```

Comment:
I don't like the idea of returning True here. It can't be False, because you raise exception if request was unsuccessful. So the returned value will always be True, and because of it we can just return None here. Also, then this method could be removed, since it's just calling another method. And the same for  `create_adult_or_young_teacher`

Reply:
I think I'll just return an integer ID then. The exception is buried a bit too deep, so it's not obvious that it will be thrown.

I prefer to keep `create_coordinator()` and `create_..._teacher()` for uniformity. It also leaves room for some additional functionality without changing the calling syntax.

Reply:
About the second, mr Knuth said: "Premature optimization is the root of all evil"

Reply:
oh come on :) I would have to make a method for creating personal info item public and I don't want that :)